xml/System.Data.Common/DbDataReader.xml

```diff
@@ -1198,27 +1164,10 @@ private static void GetDataTypes(String connectionString) {
       </Parameters>
       <Docs>
         <typeparam name="T">The type of the value to be returned.</typeparam>
-        <param name="ordinal">The type of the value to be returned.</param>
-        <param name="cancellationToken">The cancellation instruction, which propagates a notification that operations should be canceled. This does not guarantee the cancellation. A setting of <see langword="CancellationToken.None" /> makes this method equivalent to <see cref="M:System.Data.Common.DbDataReader.GetFieldValueAsync``1(System.Int32)" />. The returned task must be marked as cancelled.</param>
-        <summary>Asynchronously gets the value of the specified column as a type.</summary>
-        <returns>The type of the value to be returned.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- `T` can be one of the following types:  
-  
-|||||  
-|-|-|-|-|  
-|Boolean|Byte|Char|DateTime|  
-|DateTimeOffset|Decimal|Double|Float|  
-|Guid|Int16|Int32|Int64|  
-|String||||  
-  
- For more information, see [SqlClient Streaming Support](~/docs/framework/data/adonet/sqlclient-streaming-support.md).  
-  
- ]]></format>
-        </remarks>
+        <param name="ordinal">The zero-based column ordinal.</param>
+        <param name="cancellationToken">An optional <see cref="T:System.Threading.CancellationToken" /> to cancel the asynchronous operation.</param>
+        <summary>Asynchronously gets the value of the specified column as the requested type.</summary>
```

Comment:
Most of the async methods I've seen that receive a token parameter, do not mention the "cancellation monitoring" ability, so I agree with @divega it may not be completely necessary.  @rpetrusha do you think it would be better to add it?